src/sharedTypes.ts

```diff
@@ -1,10 +1,17 @@
 // TODO: Define transcription schema (this is just a mockup)
+export type Transcription = { words: [Word] };
+
+export type Word = {
+  word: string;
+  start_time: number;
+  duration: number;
+};
 
-import Transcription from 'sharedTypes/Transcription';
 
 export interface ProjectBase {
   schemaVersion: number;
   name: string;
+  savePath: string;
```

Comment:
Not sure if this should be stored on the project entity itself, or just part of the state that initialises when we open a file (since the project file could be moved), but this is fine for now

Reply:
Might be useful to store the prev location of export. At the very least, maybe rename to `exportPath` or `exportDefaultPath` rather than `savePath`. Might be confusing is all.